### DISCUSSION TITLE
Maximum Likelihood Estimation in Jax

### DISCUSSION THREAD

**Discussion by djlacombe** [jax-ml/jax#7385] [Show and tell] [answered]
Hello!

I wanted to make the community aware of a Jupyter notebook I uploaded to GitHub that illustrates maximum likelihood estimation of the normal linear model in Jax.

The GitHub repo is available here: [Maximum Likelihood in Jax](https://github.com/djlacombe/Maximum-Likelihood-with-Jax/blob/main/Max%20Like%20Jax.ipynb)

I just started using Jax, so there may be some errors or more efficient ways of coding all of this but I thought this might be interesting for those that use Jax.

Any comments or corrections would be appreciated.

Thanks.

**Comment by jakevdp** [comment, marked as answer]
Very cool, thanks for sharing! Related to this, one thing you may wish to check out is the recently released [jaxopt](https://github.com/google/jaxopt) package, which some members of the JAX team have been helping with. It has some interesting enhancements over the standard scipy-inspired minimization API.

**Reply by djlacombe** [reply]
Thank you in return! Again, I'm a novice Jaxer, but I'm very interested in these issues. I'd also like the notebook to be widely shared and perhaps incorporated into the docs as an example.
